libs3/README.md

```diff
@@ -0,0 +1,5 @@
+This directory contains code from the libs3 library originally written by [Bryan Ischo](https://github.com/bji).
+
+The [original libs3 project](https://github.com/bji/libs3) has been [more-or-less abandoned](https://github.com/bji/libs3/pull/70#issuecomment-414755533) and as of November 2023, the [last upstream commit](https://github.com/bji/libs3/commit/287e4bee6fd430ffb52604049de80a27a77ff6b4) was in April 2019. For a time, irods maintained [its own fork](https://github.com/irods/libs3) for use with the S3 resource plugin. However, given that it seems unlikely that Ischo will resume devleopment or maintenance of the library, the decision was made to absorb the libs3 code into the S3 resource plugin repository to ease maintenance.
```

Comment:
devleopment -> development

Reply:
irods -> iRODS

Reply:
please also reference/link the SHA at tip of the master branch for https://github.com/irods/libs3 at the time of this absorption.